commcare_connect/opportunity/views.py

```diff
@@ -639,6 +640,7 @@ def payment_delete(request, org_slug=None, opp_id=None, access_id=None, pk=None)
 def user_profile(request, org_slug=None, opp_id=None, pk=None):
     access = get_object_or_404(OpportunityAccess, pk=pk, accepted=True)
     user_visits = UserVisit.objects.filter(user=access.user, opportunity=access.opportunity)
+    user_catchments = CatchmentArea.objects.filter(opportunity_access__user=access.user)
```

Comment:
Can the filter just be `opportunity_access=access` or am I missing something?

Reply:
Fetched using opportunity_access=access.